test/ranking/ranking.test.ts

```diff
@@ -48,21 +50,53 @@ describe('ranking', () => {
 
       const specM2 = SpecQueryModel.build(
         {
-          mark: Mark.BAR,
+          mark: Mark.POINT,
           encodings: [
-            {channel: Channel.X, field: 'N', type: Type.NOMINAL},
-            {channel: Channel.Y, field: 'Q', type: Type.QUANTITATIVE},
+            {channel: Channel.X, field: 'date', type: Type.TEMPORAL},
+            {channel: Channel.Y, field: 'price', type: Type.QUANTITATIVE},
+            {channel: Channel.COLOR, field: 'symbol', type: Type.NOMINAL}
           ]
         },
         schema,
         DEFAULT_QUERY_CONFIG
       );
 
       const comparator = comparatorFactory(['aggregationQuality', 'effectiveness'], schema, DEFAULT_QUERY_CONFIG);
-      assert.isNumber(comparator(specM1, specM2));
+      assert.isTrue(comparator(specM1, specM2) > 0);
```

Comment:
I think there is `assert.lessThan` which will true better error if something is wrong.

Using `isTrue` for <, > would only produce something like `expect true to be false` when something goes wrong.